src/foremast/templates/configs/configs.json.j2

```diff
@@ -17,6 +16,7 @@
         "app_grace_period": 0,
         "max_inst": 3,
         "min_inst": 1,
+        "ssh_keypair": null,
```

Comment:
The default value should be noted here. Since it appears that `app_ssh_key` is being refactored to this key (which is good), it should follow the same behaviour and default.